packages/gas-fee-controller/src/GasFeeController.test.ts

```diff
@@ -868,7 +868,7 @@ describe('GasFeeController', () => {
       });
     });
   });
-  describe('executePoll', () => {
+  describe('_executePoll', () => {
```

Comment:
Similar for these tests.

Reply:
removed this block and moved the test into the `startPollingByNetworkClientId` test block: https://github.com/MetaMask/core/pull/1810/commits/7a404128b07d3eb446aae9a8c84004eadca3e10c

Reply:
Hmm, I see the changes for TokenListController, those look great — do we want to do the same for this controller?

Reply:
sorry whoops so in that commit I linked I moved some of the logic into the existing `startPollingByNetworkClientId` test but forgot to remove this block. That removal is done here: https://github.com/MetaMask/core/pull/1810/commits/b9502fcaf0562b6a9abab350bb6ef629e1afa128